configsrc/platform/main/vcluster.schema.json

```diff
@@ -1252,9 +1252,9 @@
           "$ref": "#/$defs/ExperimentalGenericSync",
           "description": "GenericSync holds options to generically sync resources from virtual cluster to host."
         },
-        "multiNamespaceMode": {
-          "$ref": "#/$defs/ExperimentalMultiNamespaceMode",
-          "description": "MultiNamespaceMode tells virtual cluster to sync to multiple namespaces instead of a single one. This will map each virtual cluster namespace to a single namespace in the host cluster."
+        "namespaceMode": {
+          "$ref": "#/$defs/ExperimentalNamespaceMode",
+          "description": "NamespaceMode tells virtual cluster to sync to multiple namespaces instead of a single one. This will map each virtual cluster namespace to a single namespace in the host cluster."
```

Comment:
This is not really true. It will map every namespace defined in mappings into individual namespace on host, un-mapped namespaces will follow regular single namespace behavior